csaf_2.0/register/link-relations/hash.md

```diff
@@ -0,0 +1,19 @@
+I suggest to submit for the registration of the "hash" link type the following to the relevant GitHub hosted repository [https://github.com/protocol-registries/link-relations/issues](https://github.com/protocol-registries/link-relations/issues) as issue: 
+
+Please confirm that:
+
+* [x]  You have read and understood the [requirements for registration](https://tools.ietf.org/html/rfc8288#section-2.1.1).
+* [x]  You have checked [the registry](https://www.iana.org/assignments/link-relations/) and found no current link relation types that meet your needs.
+* [x]  Your specification reference URL is stable; ideally, managed by a widely-recognised standards development organisation (e.g., published as an RFC). Otherwise, please give additional information.
+
+If so, please enter the details of the link relation type below:
+
+* Relation Name: hash
+* Description: Resource-Oriented Lightweight Information Exchange (ROLIE) is a standard to ease discovery of security content. ROLIE is built on top of the Atom Publishing Format and Protocol, with specific requirements that support publishing security content. Each ROLIE feed document MUST be a JSON file that conforms with [RFC8322]. Any existing hash file (requirement 18) MUST be listed in the corresponding entry of the ROLIE feed as an item of the array `link` having the `rel` value of `hash`.
```

Comment:
@sthagen I hope I don't over-engineer that. Looking at https://www.iana.org/assignments/link-relations/link-relations.xhtml I guess we can get away with a shorter description and broaden the scope it can be used in...
```suggestion
* Description: Refers to a resource that contains the context's hash value. The resource content SHALL start with the first byte of the hexadecimal hash value. Any subsequent data (like a filename) which is optional SHALL be separated by at least one space.
```
or
```suggestion
* Description: Refers to a resource that contains the context's hash value.
```